lib/pump.c

```diff
@@ -0,0 +1,132 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+//Read Data and convert into Struct
+const int NumSensors = 5;
+
+const int A =20;
+const int B =15;
+const int C =10;
+const int D =5;
+
+
+ struct DataStruct
+  {
+      int id;
+      int data;
+  } Datos;
+int table [5][2];
+bool PumpeisON = false;
+void ResetTable( int table[][2]){
+
+	for (int i=0;i<NumSensors;i++){
+		table[i][0]=0;
+		table[i][1]=0;
+	}
+}
+
+void printTable( int table[][2]){
+
+	for(int i=0;i<NumSensors;i++){
+		printf("id: %d value: %d \n",table[i][0],table[i][1]);
+	}
+}
+
+
+bool mainPump(struct DataStruct Datos, bool PumpeisON){
+
+    int OpenPumpe = 0;
+    int ClosePumpe = 0;
+    int SumTemp = 0;
+    int AvgTemp = 0;
```

Comment:
[not only here: ]  standard convention is that variable names start with lowercase (`openPumpe` etc)

Reply:
There is a convention we use in RIOT. While you could certainly use any convention that you like, I think it's good if one is using a certain language/framework, to use their coding conventions to make the code more homogeneous and readable.
RIOT's convention is similar to the Linux convention (with some small modifications):
- Braces for functions in a new line.
- variable_and_functions go in snake_case
- Mind the use of spaces/blank
- etc.

Reply:
Doxygen-style comments will save you (and your teammates) many headaches in the future.